cmd/main.go

```diff
@@ -52,19 +55,27 @@ func main() {
 	var (
 		metricsAddr string
 		probeAddr   string
+		logLevel    string
+		nodeName    string
```

Comment:
this is unused I think, can you add a TODO for what this is going to be for(predicate/etc..)?